src/main/java/utils/Constants.java

```diff
@@ -1,12 +1,86 @@
 package utils;
 
 public final class Constants {
-    public static final String BASE_URL = "https://petstore.swagger.io/v2";
-    public static final String CONTENT_TYPE = "application/json";
-    public static final String ACCEPT = "application/json";
     public static final int SUCCESS_STATUS_CODE = 200;
     public static final int NOT_FOUND_STATUS_CODE = 404;
+    public static final int DEFAULT_USER_ID = 2;
+    public static final int DEFAULT_USER_STATUS = 0;
+
+    public static final String BASE_URL = "https://petstore.swagger.io/v2";
+    public static final String APP_JSON_TYPE = "application/json";
+    public static final String CONTENT_TYPE_HEADER = "Content-Type";
+    public static final String ACCEPT_HEADER = "Accept";
     public static final String UNKNOWN_TYPE = "unknown";
+    public static final String PLACED_STATUS = "placed";
+    public static final String GET_METHOD = "GET";
+    public static final String POST_METHOD = "POST";
+    public static final String PUT_METHOD = "PUT";
+    public static final String DELETE_METHOD = "DELETE";
+    public static final String ERROR_TYPE = "error";
+
+    public static final String DEFAULT_USERNAME = "user_1";
+    public static final String DEFAULT_FIRST_NAME = "firstName1";
+    public static final String DEFAULT_LAST_NAME = "lastName1";
+    public static final String DEFAULT_EMAIL = "email1@gmail.com";
+    public static final String DEFAULT_PASSWORD = "qwerty";
+    public static final String DEFAULT_PHONE = "12345678";
+
+    public static final String PET_ID_FIELD = "petId";
+    public static final String QUANTITY_FIELD = "quantity";
+    public static final String SHIP_DATE_FIELD = "shipDate";
+    public static final String STATUS_FIELD = "status";
+    public static final String COMPLETE_FIELD = "complete";
+    public static final String CODE_FIELD = "code";
+    public static final String MESSAGE_FIELD = "message";
+
+    public static final String ID_FIELD = "id";
+    public static final String USERNAME_FIELD = "username";
+    public static final String FIRST_NAME_FIELD = "firstName";
+    public static final String LAST_NAME_FIELD = "lastName";
+    public static final String EMAIL_FIELD = "email";
+    public static final String PASSWORD_FIELD = "password";
+    public static final String PHONE_FIELD = "phone";
+    public static final String USER_STATUS_FIELD = "userStatus";
+
+    public static final String UPDATED_USERNAME = "updated_user_1";
+    public static final String UPDATED_FIRST_NAME = "updated_firstName1";
+    public static final String UPDATED_LASTNAME = "updated_lastName1";
+    public static final String UPDATED_EMAIL = "updated_email1@gmail.com";
+    public static final String UPDATED_PASSWORD = "updated_qwerty";
+    public static final String UPDATED_PHONE = "87654321";
+    public static final String USER_NOT_FOUND_MESSAGE = "User not found";
+
+    public static final int PET_ID = 3;
```

Comment:
Why do we need such much constants instead of just constant of the object that have constructor of all that data? Like here you can replace with:
public static final Pet DEFAULT_PET = new Pet(...);